<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSetting</class>
 <widget class="QDialog" name="ParameterSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>561</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>电机参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>连续模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="continuousPtr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>1号电机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motor_1"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>2号电机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motor_2"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>3号电机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motor_3"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>4号电机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motor_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>自动模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="autoPtr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>相机标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camAdress">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>步进模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stepPtr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>指令等待时长/ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cmdWaitTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>PLC网口参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>PLC-IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>PLC-PORT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="plc_port"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>PC-PORT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="pc_port"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="plc_ip"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>相机参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>图片保存到</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pictureSavePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="scanPath">
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color: rgb(152, 152, 152)}

QPushButton:hover{background-color: rgb(53, 161, 79);}

QPushButton:pressed{background-color: rgb(53, 161, 79)}</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParameterSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParameterSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
